VERA Programmer's Reference.md

```diff
@@ -623,6 +623,7 @@ Collisions are only detected on lines that are actually rendered.
 | 2                     | 32 px       |
 | 3                     | 64 px       |
 
+**Rendering Priority** The sprite memory location dictates the order in which it is rendered.  The sprite whose attributes are at the lowest location, $F5000, will be rendered above all other sprites; the sprite at the highest location will be rendered below all other sprites, and so forth.
```

Comment:
I think that the words "above" and "below" are misleading.  They imply the Y axis instead of the Z axis.  I think that the words "in front of" and "behind" are better.

Reply:
Yeah agreed, much better.